snapshot/lib/snapshot/simulator_launchers/launcher_configuration.rb

```diff
@@ -21,6 +21,9 @@ class SimulatorLauncherConfiguration
     attr_accessor :launch_args_set
     attr_accessor :output_directory
 
+    # xcode 9
+    attr_accessor :serialized_executions
```

Comment:
and I'd want to change this name here too, maybe make it a method `non_concurrent_simulators?`